chart/templates/replicated-supportbundle.yaml

```diff
@@ -18,6 +18,50 @@ stringData:
       name: {{ include "replicated.supportBundleName" . }}
     spec:
       collectors:
+        - exec:
+            name: replicated-app-updates
```

Comment:
thought - could we have the `name` be common between these so that the files end up in the same directory in the support bundle? any reason we might not want that?

looking at these docs: https://troubleshoot.sh/docs/collect/exec/#included-resources

Reply:
That is really good idea Craig!